bin/differential_tester_x86/LiftAndCompare.cpp

```diff
@@ -305,44 +278,42 @@ class ComparisonRunner {
                const std::vector<WhiteListInstruction> &whitelist,
                std::string_view isel_name) {
 
-    auto func1_state = (X86State *) alloca(sizeof(X86State));
-    test_runner::RandomizeState(func1_state, this->rbe);
-    func1_state->addr.ds_base.dword = 0;
-    func1_state->addr.ss_base.dword = 0;
-    func1_state->addr.es_base.dword = 0;
-    func1_state->addr.cs_base.dword = 0;
-    func1_state->aflag.af = test_runner::random_boolean_flag(this->rbe);
-    func1_state->aflag.cf = test_runner::random_boolean_flag(this->rbe);
-    func1_state->aflag.df = test_runner::random_boolean_flag(this->rbe);
-    func1_state->aflag.of = test_runner::random_boolean_flag(this->rbe);
-    func1_state->aflag.pf = test_runner::random_boolean_flag(this->rbe);
-    func1_state->aflag.sf = test_runner::random_boolean_flag(this->rbe);
-    func1_state->aflag.zf = test_runner::random_boolean_flag(this->rbe);
+    X86State func1_state = {};
```

Comment:
`= {}` is not needed, default ctor should be called even if you just do `T val;`